Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -157,6 +158,24 @@ class AppStoreConnectService {
         return Model.BetaTester(output)
     }
 
+    func inviteBetaTesterToGroups(
+        firstName: String?,
+        lastName: String?,
+        email: String,
+        groupId: String
+    ) throws {
```

Comment:
```suggestion
    func inviteBetaTesterToGroups(
        groupId: String,
        email: String,
        firstName: String? = nil,
        lastName: String? = nil
    ) throws {
```

Reply:
Reordering the parameters here make this easier to use.  Optional params should really be last.

Reply:
Alternatively:
```suggestion
    func inviteBetaTester(
        email: String,
        firstName: String? = nil,
        lastName: String? = nil,
        toGroup groupId: String
    ) throws {
```